### PR TITLE
fix: ensure font color of selected interests pill changed is white 

### LINKED ISSUE #1341: Bug: Font color of selected interests should be "White"  in Settings Page
### Describe the bug

Font color of selected interests should be "White"  in Settings Page.
Screenshot from 2023-07-08 21-01-07

### Steps to reproduce

1. Login
2. Go to User Settings

### Browsers

Chrome, Firefox

### Additional context (Is this in dev or production?)

Production

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Contributing Docs

- [X] I agree to follow this project's Contribution Docs

### PR DESCRIPTION
**Pull request by deepakrudrapaul**
## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [x] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Description

This PR fixes the font color of selected interests pill in User Settings Page
 

## Related Tickets & Documents

Fixes #1341

## Mobile & Desktop Screenshots/Recordings
Before 
Screenshot from 2023-07-08 21-01-07

After
Screenshot from 2023-07-08 21-00-45

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [x] 🙅 no documentation needed

## [optional] Are there any post-deployment tasks we need to perform?
NA

## [optional] What gif best describes this PR or how it makes you feel?
:relaxed:

### CODE REVIEW

File: components/atoms/LanguagePill/LanguagePill.tsx [hunk 1]
```diff
@@ -64,7 +64,7 @@ const LanguagePill = ({ topic, classNames, onClick }: LanguagePillProps) => {
   return (
     <div
       onClick={onClick}
-      className={`cursor-pointer text-xs text-light-slate-11 rounded-[1.875rem] w-max flex items-center gap-1 py-2 px-4 bg-light-slate-6 ${
+      className={`cursor-pointer text-xs  rounded-[1.875rem] w-max flex items-center gap-1 py-2 px-4 bg-light-slate-6 ${
```
Comment (NsdHSO) [inline components/atoms/LanguagePill/LanguagePill.tsx:67]: IMHO we need to keep this class because this component we used in multiple places.

Comment (deepakrudrapaul) [reply components/atoms/LanguagePill/LanguagePill.tsx]: Thank you for your valuable feedback! I understand your point about the Component being used in multiple places, 
 so i have used an alternative approach to fix the issue and I appreciate your warm welcome to the community.

**Review by brandonroberts — APPROVED** [review]

**Review by NsdHSO — APPROVED** [review]
LGTM

**Review by OgDev-01 — APPROVED** [review]
Thanks for pushing this fix so fast 🍕

**Pull request merged**